docs/changelog/2037.bugfix.rst

```diff
@@ -0,0 +1,2 @@
+tox config fails on `--recreate <tox-config---recreate>`_ flag, and once specified the output does not reflect the
+impact of the CLI flags - by :user:`gaborbernat`.
```

Comment:
Please rephrase this in what is fixed.

Either "tox no longer fails on..." or "fixed xyz so tox no longer..."

Reply:
This snippet shows up under the bug fixes section so I think the current wording in that context is clear to me ,🤔

Reply:
I just scanned our history and it looks like we use both styles. No hard feelings If you keep it that way, though I like the other one better 😉

Reply:
Ship it!

Reply:
I'll merge it as is, but if you put in a PR that converts it to a single format I'll accept it 🙏👍